packages/datadog-instrumentations/src/lodash.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { channel, addHook } = require('./helpers/instrument')
+
+const shimmer = require('../../datadog-shimmer')
+
+addHook({ name: 'lodash', versions: ['>=4'] }, lodash => {
```

Comment:
Do we prefer to use hooks instead the rewriter here?
wouldn't be "lighter" to use the rewriter (I guess lodash is heavily used by third party libs)?

Reply:
I used hooks instead of the rewriter approach because I did not find an effective way to do the latter. I mean, to use the rewriter I always fall in the need of requiring lodash to check the rewritten function is from it.